CONFORM_FUNCTIONS.MD

```diff
@@ -0,0 +1,299 @@
+
+One note, however, is that functions cannot be combined at this time.  That is, a `regexp` function cannot be combined with a `prefixed_number` function to operate on a single field.  
+
+## Formatting Functions
+
+There are two functions that are specifically designed to combine two or more fields into one.  
+
+### `join`
+
+The `join` function combines any number of fields using a delimiter and is useful in cases where, for example, a complex house number is stored as two separate fields in the source data.  An example of this function comes from [Honolulu, HI](https://github.com/openaddresses/openaddresses/blob/master/sources/us/hi/honolulu.json):
+
+```json
+"street": {
+	"function": "join",
+	"fields": ["HOUSEPRFX", "HOUSENUMBR"],
+	"separator": "-"
+}
+```
+
+For ease of administration, large areas of the island of Honolulu, HI [are split into nine zones and each zone is split into up to nine sections](http://westhawaiitoday.com/sections/news/local-news/big-island-street-addresses.html).  These two numbers, combined with the physical house number, create the addressable house number.  
+Understandably, the Honolulu County GIS department stores these two fields separately but for OpenAddresses purposes, they need to be combined.  For example, the source data may look like:
+
+```json
+{
+	"HOUSEPRFX": "91",
+	"HOUSENUMBR": "921"
+}
+```
+
+The addressable house number, when the `join` function above is applied, would be "91-921" for the address "91-921 Oaniani Street, Kapolei, HI".
+
+#### Definition:
+
+| parameter | value | default
+| --------- | ----- | -------
+| `function` | `join` |
+| `field` | any field name in the data source | none
+| `separator` | any string of length 0 or greater | a single space
+
+### `format`
+
+In some instances, the `join` function may not be enough to handle combining several fields where [two or more delimiters](https://github.com/openaddresses/openaddresses/blob/master/sources/nl/countrywide.json) or [hard-coded characters](https://github.com/openaddresses/openaddresses/blob/master/sources/cn/42/wuhan.json) are required for correct value expression.    
+
+An example of the `format` function can be in the [Netherlands](https://github.com/openaddresses/openaddresses/blob/master/sources/nl/countrywide.json):
+
+```json
+"number": {
+	"function": "format",
+	"fields": ["huisnummer", "huisletter", "huisnummertoevoeging"],
+	"format": "$1$2-$3"
+}
+```
+
+The positional numeric `$` format refers to the 0-based nth element in the `fields` array.  The `-` is a literal hyphen.  
+
+House numbers in the Netherlands are composed of a number concatenated with some optional letters and then a hyphen concatenated optionally with some more numbers.  Some examples of this are:
```

Comment:
I would say: "House numbers in the Netherlands are composed of a number, an optional letter, and an optional number suffix, which are concatenated using a hyphen. Some examples of this are:"